modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,49 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling mistakes.
+:keywords: autocorrect, capitalization, spell, spelling, spellcheck
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tiertwo
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
```

Comment:
```suggestion
include::partial$misc/admon-premium-plugin.adoc[]

include::partial$misc/admon-requires-6.2v.adoc[]

```
I've created this in my footnotes branch, I remember last release we faced the same issue where we all ended up making the same file. Either make the file again on this branch, exactly as it is in footnotes, or wait for footnotes docs to be merged and pull changes.